query-parser/src/find.rs

```diff
@@ -201,25 +201,21 @@ mod test_parse {
         let truncated_input = edn::Value::Vector(vec![Value::from_keyword(None, "find")]);
         assert!(parse_find(truncated_input).is_err());
 
-        let input = edn::Value::Vector(vec![
```

Comment:
I feel like our editors are going to do some battle royale at some point, but that's in the future.  Right?  Right!